client/reader/stream/empty.jsx

```diff
@@ -37,13 +37,13 @@ class FollowingEmptyContent extends React.Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<EmptyContent
-				className="is-drake"
+				className="is-following-empty"
```

Comment:
I think we can just drop the `is-` prefix now and use `stream__empty` here to match the component.